.github/workflows/ci.yml

```diff
@@ -126,33 +126,33 @@ jobs:
       env:
         GH_PAT: ${{ secrets.GITHUB_OAUTH }}
 
-  testwq:
-    runs-on: ubuntu-latest
-    needs: pre-commit
-    strategy:
-      matrix:
-        python-version: ["3.11"]
-    name: test coffea-workqueue
-
-    steps:
-    - uses: actions/checkout@v4
-    - name: Set up Conda
-      uses: conda-incubator/setup-miniconda@v2
-      env:
-        ACTIONS_ALLOW_UNSECURE_COMMANDS: true
-      with:
-        auto-update-conda: true
-        python-version: ${{ matrix.python-version }}
-        channels: conda-forge
-    - name: Test work_queue
-      shell: bash -l {0}
-      run: |
-        conda create --yes --name coffea-env -c conda-forge python=${{ matrix.python-version }} ndcctools dill conda-pack conda
-        conda activate coffea-env
-        python -m pip install --ignore-installed .
-        cd tests
-        conda-pack --output coffea-env.tar.gz
-        python wq.py coffea-env.tar.gz
+#  testwq:
```

Comment:
Not much reason to test workqueue if taskvine tests that it can eat dask graphs.
Perhapse we might want to keep some sort of smoke test for various dask backends just to ensure they play well?

Reply:
It's been clear in the past that it's nice to have coffea do its own full-stack tests for things like this.

I don't think we've achieved the level of parity where tests on dask distributed completely confirm they will work on taskvine. 

@btovar do you think that's achievable?

Reply:
Yes, definitely. I can make a pr. Further, since tasvine is orthogonal to coffea+dask, should this more in something like an examples directory?

Reply:
@btovar a test that also serves as a worked-out example would be great! I don't think it's *so* orthogonal since, for instance, dask and distributed have deeper ties than dask and taskvine (so far as I know right now), but I still test that distributed works in coffea tests for the essential ingredients.